packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -811,13 +910,15 @@ export class ContainerRuntime
 			getCompatibilityVersionDefaults(compatibilityVersion);
 
 		// The following are the default values for the options that do not affect the DocumentSchema.
-		const defaultConfigsNonVersionDependent: Required<
-			Omit<IContainerRuntimeOptionsInternal, keyof RuntimeOptionsAffectingDocSchema>
+		const defaultConfigsNonVersionDependent: Omit<
```

Comment:
@scottn12 - Are you cool with this?

Reply:
Yeah looks good! Since Jason changed `IContainerRuntimeOptionsInternal` -> `ContainerRuntimeOptionsInternal` we don't need `Required<>` anymore. Thanks for the heads up :)